src/renderer/lib/dolphin/useDolphinStore.ts

```diff
@@ -14,6 +14,8 @@ export const useDolphinStore = create(() => ({
   netplayOpened: false,
   playbackOpened: false,
   netplayDownloadProgress: null as { current: number; total: number } | null,
+  netplayDolphinVersion: "Not Found",
+  playbackDolphinVersion: "Not Found",
```

Comment:
The reason for Not Found being default is that, currently, the version only gets grabbed when dolphin is confirmed to be present from `dolphinCompleteHandler`
So, if the default was Unknown & dolphin didn't download, it would remain as unknown rather than Not found.

Think I mentioned it in another reply but also this way whilst dolphin is being downloaded it will show not found until it is downloaded I think.

Can change the defaults, but I feel like if I were to see it say "Unknown" I'd wonder why it's not recognising my dolphin rather than seeing "Not found" then I know there is no dolphin in the directory

Reply:
also I tested with a regular version of dolphin, it returns an empty string, so if someone put in a normal version of dolphin it would show unkown, as it's not a recognised slippi dolphin version

Reply:
I'm happy with this explanation.